src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
What kind of html would you like in your keys ? Something like that: `"Hello <b>world</b>"` ?
It works here, but it would most likely break the json if there are any double quote (`"`) in your html, let's say that the JSON parser escapes it, then you'd have a key like that: `"Hello <b title\"title\">world</b>"`
But then what happens in the directive when I try to find that key? I should unescape it and check against raw html. But the raw html can be transformed by angular, if you use a directive on it like *ngIf or something like that.
Also angular adds custom attributes on all elements for scoped styles, and then your original html would look like this: `Hello <b _ngcontent-bar-15>world</b>` and there's no way I could match it to your key.

So now, the extraction tool should not pick up any html :)